tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java

```diff
@@ -91,65 +91,97 @@ public void testDeleteOffsetsNonExistingGroup() {
 
     @ClusterTest
     public void testDeleteOffsetsOfStableConsumerGroupWithTopicPartition() {
-        for (Map<String, Object> consumerConfig : consumerConfigs) {
-            createTopic(TOPIC);
-            testWithConsumerGroup(TOPIC, 0, 0, Errors.GROUP_SUBSCRIBED_TO_TOPIC, true, consumerConfig);
-            removeTopic(TOPIC);
+        int idx = 0;
+        for (Iterator<Map<String, Object>> it = consumerConfigs.iterator(); it.hasNext(); idx++) {
+            Map<String, Object> consumerConfig = it.next();
+            String topic = TOPIC_PREFIX + idx;
+            String group = GROUP_PREFIX + idx;
+            createTopic(topic);
+            testWithConsumerGroup(topic, topic, group, 0, 0, Errors.GROUP_SUBSCRIBED_TO_TOPIC, true, consumerConfig);
+            removeTopic(topic);
         }
     }
 
     @ClusterTest
     public void testDeleteOffsetsOfStableConsumerGroupWithTopicOnly() {
-        for (Map<String, Object> consumerConfig : consumerConfigs) {
-            createTopic(TOPIC);
-            testWithConsumerGroup(TOPIC, -1, 0, Errors.GROUP_SUBSCRIBED_TO_TOPIC, true, consumerConfig);
-            removeTopic(TOPIC);
+        int idx = 0;
+        for (Iterator<Map<String, Object>> it = consumerConfigs.iterator(); it.hasNext(); idx++) {
+            Map<String, Object> consumerConfig = it.next();
+            String topic = TOPIC_PREFIX + idx;
```

Comment:
Maybe we can use test case + protocol name to be unique name

Reply:
Yes, thanks for the suggestion. Updated it.